src/components/RecNode/RecNode.tsx

```diff
@@ -41,10 +42,24 @@ const RecNode: FC<RecNodeProps> = ({ pathwayState, isActionable }) => {
       {isActionable && (
         <form className={classes.commentsForm}>
           <label>Comments:</label>
-          <button className={indexClasses.button} onClick={(e): void => e.preventDefault()}>
+          <button
+            className={indexClasses.button}
+            onClick={(e): void => {
+              e.preventDefault();
+              setComments(
+                comments +
+                  // eslint-disable-next-line max-len
```

Comment:
Instead of disabling this eslint rule can you break this string over multiple lines?  Although I hate the look I think it's better we try to avoid selectively turning off the linter.

Use `+` or `\` (see [`Long literal strings` section in MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String))